target/linux/ath79/files/drivers/net/ethernet/atheros/ag71xx/ag71xx_main.c

```diff
@@ -1587,6 +1592,10 @@ static int ag71xx_probe(struct platform_device *pdev)
 		ag->pllregmap = NULL;
 	}
 
+	if (of_property_read_bool(np, "builtin-switch")) {
```

Comment:
Maybe we can check `fixed-link` node instead of a new property?
ref: https://www.kernel.org/doc/html/latest/devicetree/kernel-api.html#c.of_get_child_by_name